terraform/aws/main.tf

```diff
@@ -57,16 +67,24 @@ module "eks" {
     }
   }
 
+  eks_managed_node_group_defaults = {
+    disk_size = 100
+  }
+
   eks_managed_node_groups = {
     main = {
-      min_size = 3
-      max_size = 3
-      desired_size = 3
-      instance_types = ["t3.xlarge"]
+      min_size = 2
+      max_size = 2
+      disk_size = 100
+      desired_size = 2
+      ami_type = "AL2_x86_64"
+      ami_id = data.aws_ami.current_aws_ami.id
+      enable_bootstrap_user_data = true
+      instance_types = ["g4dn.xlarge"]
```

Comment:
Converted to draft again to introduce a mechanism for setting instance type for non-GPU clusters